tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java

```diff
@@ -54,7 +54,7 @@
   private static final int BLOCK_SIZE = 1024;
   @ClassRule
   public static LocalAlluxioClusterResource sLocalAlluxioClusterResource =
-      new LocalAlluxioClusterResource(100000000, BLOCK_SIZE,
```

Comment:
I think this should be `100 * Constants.MB`. I guess this is why it's nicer to use the predefined sizes instead of zeros :)